build-logic/performance-testing/src/main/groovy/gradlebuild/performance/generator/tasks/RemoteProject.groovy

```diff
@@ -25,11 +25,13 @@ import org.gradle.api.tasks.Input
 import org.gradle.api.tasks.Optional
 import org.gradle.api.tasks.OutputDirectory
 import org.gradle.api.tasks.TaskAction
+import org.gradle.work.DisableCachingByDefault
 
 /**
  * Checkout a project template from a git repository.
  */
 @CompileStatic
+@DisableCachingByDefault(because = "Not worth caching")
 abstract class RemoteProject extends DefaultTask {
```

Comment:
Wouldn't this make sense to cache locally only, if we had the feature?

Reply:
Maybe. I suppose up-to-date checks would be good enough here.